plugin-scalebar/src/main/java/com/mapbox/maps/plugin/scalebar/ScaleBarImpl.kt

```diff
@@ -114,7 +114,7 @@ class ScaleBarImpl : ScaleBar, View {
       strokePaint.textSize = value.textSize
       scaleTable = if (value.isMetricUnits) metricTable else imperialTable
       unit = if (value.isMetricUnits) METER_UNIT else FEET_UNIT
-      strokePaint.strokeWidth = value.textBorderWidth
+      strokePaint.strokeWidth = if (value.showTextBorder) value.textBorderWidth else 0F
```

Comment:
is this related to text being cut fix?

Reply:
Yes, if we want to render border around the text we need to take it into account, otherwise we set it to 0 so we avoid an `if` check on every draw. See code a few lines below:
```Kotlin
val textMaxRightPx = x + strokePaint.strokeWidth + when (textPaint.textAlign) {
```